### ISSUE TITLE
Unpublish permission is ignored when viewing content

### ISSUE THREAD

**Issue by digbyswift** [umbraco/Umbraco-CMS#12910]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.4.3+

### Bug summary

If a user has *only* permission to browse content and unpublish, the ability to unpublish content is not available to that user.

### Steps to reproduce

1. Create a User group that has only Browse content permission and also Unpublish permission.
2. Allow this new group access to the root content tree
3. Assign a user to just this group
4. Browse to any published content
5. The user will not be able to unpublish the content. Where the save/publish button would normally be, there is only a "Save and preview" option.

### Expected result / actual result

When browsing published content I would expect to see the usual green button present but with Unpublish as the only option.

If the content is not published, I would not expect to the see the Unpublish button.

**Comment by nul800sebastiaan** [comment]
I think they need publish permissions to unpublish, could you check if that helps?

**Comment by digbyswift** [comment]
@nul800sebastiaan That is correct, but then that also gives the ability to Publish which isn't needed. I should be able to Unpublish without having to Publish. The use case is for a group of specific users to be able to housekeep content. We don't want that group to be able to publish unpublished content.

Ideally, one permission should not be linked to the other (my preference), or the UI should implement a disable/toggle to indicate the relationship.

**Comment by nul800sebastiaan** [comment]
Yeah I agree it's a bit weird, I'll have to ask around why it is this way.

In the mean time, you can implement a publishing notificationhandler that will cancel any publishing someone in the role tries to do: https://our.umbraco.com/Documentation/Fundamentals/Code/Subscribing-To-Notifications/

**Comment by kjac** [comment]
Hi @digbyswift, 

Looks like this is indeed a bug. The UI shouldn't show "Save and preview..." if the editor has no "Update" permission; in your setup, the only button that should be shown is "Unpublish".

![image](https://user-images.githubusercontent.com/7405322/207303485-fa95c35f-2aea-405f-9269-7197471d1b0f.png)

...one could argue if it should be styled as big and green, as unpublishing likely isn't a desired action as such. Perhaps it should be default grey-ish. But that's an implementation detail.

We'd love some help getting this fixed, so the editor buttons actually reflect the permitted abilities of any given user - no more, no less.

I reckon it has to be solved somewhere around these parts: https://github.com/umbraco/Umbraco-CMS/blob/v11/contrib/src/Umbraco.Web.UI.Client/src/common/services/contenteditinghelper.service.js#L218

**Comment by MaxiveLegend** [comment]
Just came across this issue today in Umbraco v13.14.0. I can't immediately check if this is also relevant for newer versions, but I will check that sometime in the near future and report back for umbraco v17 when I get the chance.

**Comment by nielslyngsoe** [comment]
Hi @MaxiveLegend let us know how you experience this particular issue in v.17, as that would be the version for us to fix this in, as v.13 is in security phase. Thanks in advance, please tag me when you have done the investigation.

**Comment by MaxiveLegend** [comment]
Hi @nielslyngsoe , apologies it took me a while to circle back on this. Work has been busy 😜️
I have just verified that this does not seem to be an issue in V17. My guess is that something in the bellissima migration has fixed this issue after V13.
I'm currently working on a fix for this in V13, and will link the PR for that here (and vice versa) when I'm done.

**Comment by AndyButland** [comment]
Thanks for getting back @MaxiveLegend.  I will close this now then.  As Niels mentioned, 13 is in security phase now so we aren't making or accepting PRs for bug fixes on that version any more.
